Source/WebKit/NetworkProcess/Notifications/NetworkNotificationManager.cpp

```diff
@@ -230,5 +231,12 @@ void NetworkNotificationManager::getPermissionStateSync(WebCore::SecurityOriginD
     getPushPermissionStateImpl(m_connection.get(), WTFMove(origin), WTFMove(completionHandler));
 }
 
+const std::optional<SharedPreferencesForWebProcess> NetworkNotificationManager::sharedPreferencesForWebProcess(const IPC::Connection& connection) const
```

Comment:
Returning const object is useful if it returns reference or pointer; here it return a copy so `const` should not be necessary:
`std::optional<SharedPreferencesForWebProcess> NetworkNotificationManager::sharedPreferencesForWebProcess(const IPC::Connection& connection) const`